compiler/rustc_feature/src/active.rs

```diff
@@ -397,8 +397,6 @@ declare_features! (
     (active, if_let_guard, "1.47.0", Some(51114), None),
     /// Allows using imported `main` function
     (active, imported_main, "1.53.0", Some(28937), None),
-    /// Allows in-band quantification of lifetime bindings (e.g., `fn foo(x: &'a u8) -> &'a u8`).
-    (active, in_band_lifetimes, "1.23.0", Some(44524), None),
```

Comment:
Features are usually moved from the `active` list to the `removed` list instead of being removed entirely.